src/util/et/error_message.c

```diff
@@ -69,9 +69,9 @@ void com_err_terminate(void)
         enext = e->next;
         free(e);
     }
+    et_list = NULL; terminated = 1;
```

Comment:
Library calls should be coded with respect to safe coding practices so static pointers should be NULLed to be able to test it for NULL to avoid it to point on already freed object.
Also the critical section should be mutexed all not only the part of it.

Reply:
My point here and in the next comment is that `terminated` is not considered part of the critical section.  (It's a debugging aid, as the comment at its definition indicates.)  Your change on line 284 is therefore unsafe, as it happens unsynchronized.

Also, statements still each get their own line.